.circleci/config.yml

```diff
@@ -0,0 +1,69 @@
+---
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: debian:jessie
```

Comment:
You can probably use `image: 853032795538.dkr.ecr.us-east-1.amazonaws.com/frontrow/circleci-haskell-build:latest` and leave a comment explaining that it is the `frontrow` base build image for Haskell CircleCI builds. Then you don't have to do `apt-get` or install `stack`.

Reply:
I thought about that, but decided against it because I didn't want to deal with authenticating against our image repository.

Reply:
Ah damn I thought this was public for some reason. No worries.